### DISCUSSION TITLE
Approval process for RFCs

### DISCUSSION THREAD

**Discussion by CalvinRodo** [cds-snc/docs#11] [General]
So #8 has been approved however I'm hesitant to merge it as I'm not sure it will actually be adopted by CDS. I think we need to come up with an agreed upon process for the approval of RFCs before we merge in. 

For instance should the approval of @wmoussa-cds be required before an RFC is considered accepted so that we have the support of management? Should @maxneuvians need to sign off as this RFC puts ownership of the label syncing actions in the Internal SRE team. 

#8 has an impact on other communities that haven't agreed to the RFC process in particular the Product Management Community and so without that communities stamp of approval I think we would be hard pressed to implement that recommendation. It also has impacts on the various business units as well and none of them are aware of this. 

Regardless of all that I'm also uneasy about having just two members of the dev community signing off on something that impacts the entire community. While I don't think we need approval from everyone I think to ensure success we at least want a few members to approve before we commit to something.

**Comment by dj2** [comment]
I think part of this is how I've historically used the approval button. I use it as, this looks good to me, when it's ready I'm happy if you land it. As opposed to go ahead and land this now. So, I probably need to stop doing that and just add comments when something LGTM.

For something like https://github.com/cds-snc/docs/pull/8 we probably want to add someone from the various impacted communities to get agreement the direction is correct. So, probably, PM, SRE and post something about it in the dev slack channel.

As well, even with my approval, I think it should wait the 2 week period we talked about before landing to give folks chime in if they so desire.

**Reply by CalvinRodo** [reply]
So I think next step then is an RFC outlining the RFC Approval process

**Reply by dj2** [reply]
Done: https://github.com/cds-snc/docs/pull/12
